emily/handler/src/database/accessors.rs

```diff
@@ -460,16 +473,41 @@ pub async fn add_chainstate_entry(
             .await
             .and_then(|existing_entry: ChainstateEntry| {
                 if &existing_entry != entry {
+                    debug!("Inconsistent state because of a conflict with the current interpretation of a height.");
+                    debug!("Existing entry: {existing_entry:?} | New entry: {entry:?}");
                     Err(Error::from_inconsistent_chainstate_entry(existing_entry))
                 } else {
                     Ok(())
                 }
             });
 
-    // Exit here unless this chain height hasn't been detected before.
     match current_chainstate_entry_result {
         // Fall through if there is no existing entry..
         Err(Error::NotFound) => (),
+        // If the chainstate entry is already in the table but the api believes the chaintip is behind
+        // this entry, that means a reorg has occurred and the api got pulled back, but then it went
+        // back to the chain it had been following before the reorg. This is a stable state, and we
+        // will skip putting it into the table but will update the api state.
+        Ok(_) if api_state.chaintip().key.height < entry.key.height => {
```

Comment:
No, because the data for those requests isnt "disabled" - they're deleted. https://github.com/stacks-network/sbtc/blob/26b306817249879e5967a13ebcbffbe0c25804b5/emily/handler/src/database/entries/deposit.rs#L165